src/TouristInfoCards.js

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+
+const TouristInfoCards = () => {
```

Comment:
I'm not sure if you talked about this in Group A, but in Group B we had a discussion about how it may be useful to further split up this component to show an individual card that uses props to control what the card shows.

This way, we can re-use that component all over the application, instead of always having to render multiple cards.